### ISSUE TITLE
Needs License

### ISSUE THREAD

**Issue by LeifAndersen** [czchapma/onlineharassment#1]
I don't really see a license file attached to this project. Its probably a good idea to add one before the project gets too big.

Also hi everyone, I'm Leif. :D Thank you so much for making this. <3

**Comment by czchapma** [comment]
Yes. This has been on my to-do list.  Thank you for the reminder, would
love your opinion on what license would be best.

**Comment by LeifAndersen** [comment]
Honestly, it depends on what your goals are for the project. Generally, if you want to make sure the project stays open source, you'd go with some sort of copyleft license. Whereas, if you want to encourage as many people as possible to contribute, you'd go with a permissive license.

Since I have a suspicion you're going to go for the latter (although I could absolutely be wrong about that), your choices are between something like the MIT/Expat license or the Apache v2 license. The former license has the advantage that it is significantly simpler, being only 3 or so paragraphs, while the apache v2 one is significantly larger. However, the apache v2 license has other advantages, such as an implicit patent grant if you decide to patent any of your techniques.

Either way though, since there is no license at the moment, you will need the approval of everyone who has contributed thus far (or at least everyone who owns code in the repo).

I hope that helps. :D

**Comment by redSlug** [comment]
I'm so glad this project is moving forward :) i am down with whatever
license you choose

**Comment by LeifAndersen** [comment]
I generally air on the side of caution, so I would go with Apache v2. But
we can talk about this in person today as well.

**Comment by LeifAndersen** [comment]
Okay, PR #2 attaches the apache v2 license. You can merge it in if everyone who has contributed agrees. I can alternatively attach a different license if you'd rather.
